src/renderer/features/currency/CurrencyForm/ui/CurrencyForm.tsx

```diff
@@ -77,7 +81,7 @@ export const CurrencyForm = ({ onSubmit }: Props) => {
         onChange={({ value }) => currency?.onChange(value.id)}
       />
 
-      <Button className="w-fit ml-auto" type="submit">
+      <Button className="w-fit ml-auto" type="submit" disabled={!currency?.isDirty && !fiatFlag?.isDirty}>
```

Comment:
UI should not know anything about business relations between fields
Computation for this flag should stay in the model
```javascript
const isFormValid = combine(...);
```